recipes/mdal-python/bld.bat

```diff
@@ -0,0 +1,2 @@
+set CMAKE_GENERATOR=Ninja
```

Comment:
The comments in the `build.sh` likely also apply here.

Reply:
Intetesting and reasonable comment - as with most of the ones below I actually just copied the basic structure of the recipe from the working pdal-python recipe and then adapted it. This kind of came with it - but lets try without and see what happens